ui/raidboss/data/05-shb/ultimate/the_epic_of_alexander.js

```diff
@@ -450,10 +453,10 @@ let getHeadmarkerId = (data, matches) => {
     {
       id: 'TEA Fate Alpha Phase',
       regex: Regexes.startsUsing({ source: 'Perfect Alexander', id: '487B', capture: false }),
-      regexCn: Regexes.startsUsing({ source: '完美亚历山大', id: '487B', capture: false }),
       regexDe: Regexes.startsUsing({ source: 'Perfekter Alexander', id: '487B', capture: false }),
-      regexFr: Regexes.startsUsing({ source: 'Alexander parfait', id: '487B', capture: false }),
+      regexFr: Regexes.startsUsing({ source: 'Alexander Parfait', id: '487B', capture: false }),
```

Comment:
Please, please, stop making these changes.  I've asked you before so many times and closed a pull request that did just this.  I don't think I can communicate this more clearly.

 I appreciate that you want to change these, but it is not worth your time to edit them or my time to review your edits.  Capitalization in logs is not always consistent and so it is not even clear to me that these changes are even always correct.

I am not trying to be rude and I mean this as kindly as possible, but please stop.  I am not going to merge any more patches with capitalization changes in them, including this one.